### DISCUSSION TITLE
Any chance for a standalone version?

### DISCUSSION THREAD

**Discussion by rs-sim-x** [AIrjen/OneButtonPrompt#114] [Ideas]
would love a version to take with me and work on prompts while I am waiting for the train for example. would this be possible somehow with a html  page sorts of that just puts out the prompts, I would copy and paste them elsewhere to refine them manually. am aware it defeats a bit the "suprise" purpose of the extension and was wondering if this is possible. one case where I really dread having no coding skills :(.

thanks for maybe considering this.

**Comment by AIrjen** [comment]
I am considering the option to export prompts to file! But then it needs an interface as well… or create a simple batch file

**Reply by Reasonant** [reply]
You could remove the api integration, so it doesn't need the automatic1111 to work. Then use gradio as interface.

**Comment by AIrjen** [comment]
Hi, I finally got around building a basic website for One Button Prompt that can do all the prompt generation:

https://airjen.pythonanywhere.com/

Might add more there later, but it should satisfy your requirements.
